components/HomeHeader.vue

```diff
@@ -179,137 +67,59 @@ export default Vue.extend({
 
       this.relatedItems[key] = itemData;
     }
-    this.loading = false;
+
+    if (this.items.length === 0) {
+      this.extraText = this.$t('homeHeader.welcome.noItems').toString();
+    } else {
+      this.extraText = this.$t('homeHeader.welcome.checkNewItems').toString();
+    }
+    window.setTimeout(this.hideWelcomeMessage, 1500);
   },
   methods: {
-    ...mapActions('playbackManager', ['play']),
-    getRelatedItem(item: BaseItemDto): BaseItemDto {
-      const rItem = this.relatedItems[this.items.indexOf(item)];
-      if (!rItem) {
-        return item;
-      }
-      return rItem;
-    },
-    getOverview(item: BaseItemDto): string {
-      if (item.Overview) {
-        return this.sanitizeHtml(item.Overview);
+    hideWelcomeMessage(): void {
+      if (this.items.length === 0) {
+        const elem = this.$refs.headerWelcome as HTMLElement;
+        // As the height of the element varies as we're using automatic values, we set first
+        // the screen height of the element, so the animation plays correctly when the height is set to 0
+        elem.style.maxHeight = elem.scrollHeight + 'px';
+        elem.classList.add('no-items');
```

Comment:
Use bound classes/styles for this.

Reply:
@MrTimscampi Although I agree that we should bind this kind of stuff as much as possible, I don't think here it makes sense. It adds overcomplication (we need to add another boolean variable to the equation, as ``!items.length && !loading`` contradicts each other):

* ``items.length`` will always be 0 when there are no items to show, **but there's a sensible amount of seconds that it's 0 while data is being fetched from server, making the content start the animation and strop halfway**
* Setting ``loading`` to false will immediately kill the Welcome component, as it will also trigger the ``v-if``
* Getting the element's scrollHeight will need a computed property (which are a few lines) while this is a single one.